src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -110,22 +109,23 @@
   {:flex-direction "column"})
 
 (def popular-list-chat-action
-  {:background-color "rgb(220, 214, 251)"
+  {:background-color "rgba(67, 96, 223, 0.2)"
```

Comment:
Can we extract this in `commons` ?

Reply:
and also if 0.2 is opacity better put it in :opacity, and convert the rgb to color code

Reply:
done in separate PR in develop - without opacity though so might have to be adjusted